File: logic-and-algorithms/C/Exercicios/retangulo/retangulo.c
#include <math.h>
#include <stdio.h>
int main(int argc, char const *argv[])
{
    double base,altura,area,perimetro,diagonal;
    printf("Digite o valor da base do retangulo: ");
    scanf("%lf",&base);
    printf("Digite o valor da altura do retangulo: ");
    scanf("%lf",&altura);
    area = base * altura;
    perimetro = 2 * base + 2 * altura;
    diagonal = sqrt(pow(base,2.0) + pow(altura,2.0));
    printf("Area: %.2lf\n",area);
    printf("Perimetro: %.2lf\n",perimetro);
    printf("Diagonal: %.2lf\n",diagonal);

    return 0;
}

File: Algoritmos e logica Curso/Linguagem Python/Exercicios/Retangulo.py
import math
Base = float(input("Digite a Base do seu retângulo!"))
altura = float(input("Digite a altura do seu retângulo!"))
def Area(Base, altura) : 
    Area = Base * altura
    return Area
def Diagonal(Base,altura):
    Diagonal = math.sqrt(( Base ** 2) + (altura ** 2))
    return Diagonal
def perimetro(Base,altura):
    perimetro = (2 * Base) + (2* altura)
    return perimetro
print(f"Area: {Area(Base,altura)}")
print(f"Diagonal: {Diagonal(Base,altura)}")
print(f"perimetro: {perimetro(Base,altura)}")